pom.xml

```diff
@@ -377,38 +377,30 @@
 
         <plugins>
             <plugin>
-                <groupId>org.kaazing</groupId>
+                <groupId>org.codehaus.mojo</groupId>
                 <artifactId>license-maven-plugin</artifactId>
-                <version>1.0.0.11</version>
+                <version>1.10</version>
+                <configuration>
+                    <outputDirectory>target</outputDirectory>
+                    <thirdPartyFilename>NOTICE.txt</thirdPartyFilename>
+                    <fileTemplate>org/kaazing/community/notice/third-party-template.ftl</fileTemplate>
+                </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.kaazing</groupId>
+                        <artifactId>community.notice</artifactId>
+                        <type>jar</type>
+                         <version>develop-SNAPSHOT</version>
```

Comment:
The version should be changed after merging [PR](https://github.com/kaazing/community/pull/44) and releasing community.